P0009/P0009.bs

```diff
@@ -978,26 +1034,35 @@ template<ptrdiff_t... LHS, ptrdiff_t... RHS>
 
 1. A *layout mapping policy* is a class that contains a *layout mapping*, a nested class template.
 
-2. A *layout mapping policy* shall meet the requirements in table �.
+2. A *layout mapping policy* meets the requirements in Table �.
 
-3. A *layout mapping* shall meet the requirements of `DefaultConstructible`, `CopyAssignable`, `EqualityComparable`, and the requirements in table �.
+3. A *layout mapping* meets the requirements of *Cpp17DefaultConstructible*, *Cpp17CopyAssignable*, *Cpp17EqualityComparable*, and the requirements in Table �.
 
 4. In Table �:
     * `MP` denotes a layout mapping policy.
     * `E` denotes a specialization of `extents`.
-    * `e` denotes an object of type `E` defining a domain multi-index space.
+    * `e` denotes an object of type `E` defining a multidimensional index space.
     * `r` is a value of an integral type such that 0 <= `r` < `e.rank()`.
-    * `i...` and `j...` are packs of an integer type denoting values in the multi-index space `e`, the `r`*th member of packs `i...` and `j...` are denoted by `i[r]` and `j[r]`, and `sizeof...(i)==E::rank()`, 0 <= `i[r]` < `e.extent(r)`, `sizeof...(j)==E::rank()`, and 0 <= `j[r]` < `e.extent(r)`.
+    * If `k...` is a pack of an integer type, we say that "`k...` is in `e`" 
+      if `k...` denotes a multi-index in `e`.
+      Let the packs of integer type `i...` and `j...` be in `e`.
+      Denote the <math>r</math>-th member of packs `i...` and `j...` by 
+      <math>i<sub>r</sub></math> and <math>j<sub>r</sub></math>, respectively. 
+      Let `r` equal <math>r</math>.
+      `sizeof...(i)==E::rank()` is `true`, 
```

Comment:
Not sure why we say "is `true`" here, but if that's an LWG thing it's fine.

Reply:
@dhollman This is an LWG thing.  Boolean code-font statements are just statements that could evaluate to `false` or `true`.

Reply:
It's different when something Returns a `bool`; then it just returns the value of the code-font `bool` statement.